### PR TITLE
Edit for grammar and style

### PR DESCRIPTION
**Pull request by john-andrilla**
This topic is a good place to explain how to get users set up correctly before they get started using DeepNumPy. Tell what general knowledge is expected-though you did touch on that already. Tell what environment or other packages or frameworks are required before moving on. Python only? Include links to high-level topics as necessary.

### CODE REVIEW

**Review by aaronmarkham — COMMENTED** [review]
This is fine, but I think the names need to be updated to MXNet/NumPy or MXNet-NumPy instead of DeepNumPy.
@reminisce ?

**Comment by reminisce** [comment]
Yes, we need to change the name to `MXNet/np`.

File: guide/crash-course/index.rst [hunk 1]
```diff
@@ -1,10 +1,8 @@
-Crash Course
+Getting started with DeepNumPy
 ============
 
-This crash course will give you a quick overview of the core concept of DeepNumPy
-(manipulating multiple dimensional arrays) and Gluon (create and train neural
-networks). This is a good place to start if you are already familiar with
-machine learning or other deep learning frameworks.
+This crash course shows how to get started with DeepNumPy on MXNet. The topics here provide a quick overview of the core concepts for both DeepNumPy, which helps you manipulate multiple dimensional arrays, and Gluon, which helps you create and train neural
```
Comment (john-andrilla) [inline guide/crash-course/index.rst]:
```suggestion
This crash course shows how to get started with NumPy on MXNet. The topics here provide a quick overview of the core concepts for both NP on MXNet, which helps you manipulate multiple dimensional arrays, and Gluon, which helps you create and train neural
```

File: guide/crash-course/index.rst [hunk 2]
```diff
@@ -1,10 +1,8 @@
-Crash Course
+Getting started with DeepNumPy
```
Comment (john-andrilla) [inline guide/crash-course/index.rst]:
```suggestion
Getting started with NumPy on MXNet
```

**Comment by reminisce** [comment]
@aaronmarkham Is this good for merge? Thanks.

**Review by aaronmarkham — APPROVED** [review]

**Pull request merged**